packages/chart/src/components/LineChart/components/LineChart.BumpCircle.tsx

```diff
@@ -0,0 +1,98 @@
+import { useState } from 'react'
+import { Group } from '@visx/group'
+import { type Column } from '@cdc/core/types/Column'
+import '../index.scss'
```

Comment:
Can you move the index.scss file into this directory and rename it to something like linechart-bumpcircle.styles.css to keep the styles closer to the component if its only being used in this file?

Reply:
Removed the file. I was originally going to implement then realized I didn't need it and forgot to remove